packages/expo/src/Expo.fx.tsx

```diff
@@ -14,6 +15,26 @@ if (isRunningInExpoGo()) {
   ErrorUtils.setGlobalHandler(createErrorHandler(globalHandler));
 }
 
+// Warn if the New Architecture is not explicitly enabled in the app config and we are running in Expo Go.
+// This could be problematic because you will be developing your app with the New Architecture enabled and
+// but your builds will have the New Architecture disabled.
```

Comment:
```suggestion
// Warn if the New Architecture is not explicitly enabled in the app config and we are running in Expo Go.
// This could be problematic because you will be developing your app with the New Architecture enabled
// but your builds will have the New Architecture disabled.
```